test.js

```diff
@@ -0,0 +1,32 @@
+// Instantiate a new instance of the infobox
```

Comment:
Is the intention to leave this in the repo or is this just a convenience for development? Would be good to turn it into an actual unit test.